pkg/ipamd/ipamd.go

```diff
@@ -831,7 +872,8 @@ func (c *IPAMContext) tryAssignIPs() (increasedPool bool, err error) {
 			err = c.awsClient.AllocIPAddresses(eni.ID, 1)
 			if err != nil {
 				ipamdErrInc("increaseIPPoolAllocIPAddressesFailed")
-				return false, errors.Wrap(err, fmt.Sprintf("failed to allocate one IP addresses on ENI %s, err: %v", eni.ID, err))
+				log.Errorf("failed to allocate one IP addresses on ENI %s, err: %v", eni.ID, err)
```

Comment:
seems this error is logged in multiple times.
If we use errors.As as i commented above, we don't need to change this place.

Reply:
Another reason why I changed this is because we are logging it multiple times in the file.